src/lib/components/workspace/workspace-tab/myWorkspace.svelte

```diff
@@ -80,14 +146,68 @@
       email = value?.email;
     }
   });
+  const getUserDetails = async () => {
+    const response = await _viewModel.getUserDetailsOfWorkspace(
+      currentWorkspaceDetails.id,
+    );
+    if(response.data && response.data.data){
+    users = response.data.data;
+    const indexToRemove = users.findIndex(
+      (dataObj: any) => dataObj.email === email,
+    );
+    loggedInUser = users.splice(indexToRemove, 1)[0];
+    users.sort((a, b) => a.role.localeCompare(b.role));
+    return users;
+  }
+  };
+
+  const activeWorkspaceSubscribe = activeWorkspace.subscribe(
+    (value: WorkspaceDocument) => {
+      if (value) {
+        currentWorkspaceDetails.id = value._data._id;
+        currentWorkspaceDetails.name = value._data.name;
```

Comment:
use spread operator to add values like 
currentWorkspaceDetails = {...currentWorkspaceDetails, id=value._data._id, name=value._data.name};

Reply:
use this when u have to add multiple values one after another